### ISSUE TITLE
As data user i could not send my shopping cart to the DLP

### ISSUE THREAD

**Issue by UteH** [dzhw/metadatamanagement#2759]
https://git.cowain.team/dzhw/dlp/-/issues/166#note_2749

DLP are already fixing the error handling from DLP perspective. Still, we wanted to investigate how that happend.

> OK, here's what's going wrong:
> When the data user sent the application, for a reason unknown to me at the moment, the MDM did not provide an answer.

**Comment by UteH** [comment]
@rreitmann suggested to test the following scenario

> Vermutung ist also:
>
> 1. MDM leitet Warenkorb "xyz" an DLP weiter
> 2. User bearbeitet Warenkorb "xyz" im MDM (im anderen Tab)
> 3. DLP versucht "veralteten" Warenkorb "xyz" zu speichern -> Fehler (bearbeitet)

**Comment by UteH** [comment]
1. I added EUROGRADUATE to the shopping card, ordered it an was forwared to the DLP -> only EUROGRADUATE in the DLP 
1. I edited the shopping cart in a separate tab: I added DZHW Graduate Panel 2005
1. I ordered EUROGRADUATE and DZHW Graduate Panel 2005 and was forwarded to the DLP -> still only EUROGRADUATE in the DLP 
2. I clicked on "edit shopping cart" in the DLP and I am forwarded to the MDM -> both data packages are in the shopping cart
3. now, when I am in the DLP, proceed with only EUROGRADUATE and go back to tab "choose data" in the application process -> both data packages are listed!
